tamr_client/dataset/unified.py

```diff
@@ -0,0 +1,106 @@
+"""
+See https://docs.tamr.com/reference/dataset-models
+"""
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+from tamr_client import response
+from tamr_client.instance import Instance
+from tamr_client.project import Project
+from tamr_client.session import Session
+from tamr_client.types import JsonDict
+from tamr_client.url import URL
+
+
+class DatasetNotFound(Exception):
```

Comment:
Already in the `dataset/unified` namespace, so this should just be named "NotFound":

```suggestion
class NotFound(Exception):
```